.gitignore

```diff
@@ -1 +1,2 @@
 #Files to ignore
+test.md
```

Comment:
- [ ] [OPTIONAL] Kindly add `node_modules` to your `.gitignore` file. This will prevent git from tracking your node modules folder when and if you have set up linters locally.